### PR TITLE
Fix broken component example links

### PR DESCRIPTION
**Pull request by seanpcoyle**
## Please verify the following:

- [ ] `yarn test` **jest** tests pass with new tests, if relevant
- [ ] `yarn lint` **eslint** checks pass with new code, if relevant
- [ ] `yarn format:check` **prettier** checks pass with new code, if relevant
- [ ] `README.md` (or relevant documentation) has been updated with your changes
- [ ] If this affects functionality there aren't tests for, I manually tested it, including by generating a new app locally if needed ([see docs](https://docs.infinite.red/ignite-cli/contributing/Contributing-To-Ignite/#testing-changes-from-your-local-copy-of-ignite)).

## Describe your PR

This fixes broken links in the boilerplate components.

### CODE REVIEW

File: boilerplate/app/components/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export interface BaseToggleInputProps<T> {
 /**
  * Renders a boolean input.
  * This is a controlled component that requires an onValueChange callback that updates the value prop in order for the component to reflect user actions. If the value prop is not updated, the component will continue to render the supplied value prop instead of the expected result of any user actions.
- * @see [Documentation and Examples]{@link https://docs.infinite.red/ignite-cli/boilerplate/components/Toggle/}
+ * @see [Documentation and Examples]{@link https://docs.infinite.red/ignite-cli/boilerplate/app/components/Toggle/}
```
Comment (frankcalise) [inline boilerplate/app/components/Toggle/Toggle.tsx:121]: Actually this is not a doc page, we'd need to move this comment into the Checkbox, Radio and Switch components that fall under the Toggle directory

**Review by frankcalise — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Great find, would you mind just helping out touch up the Toggle child components that are missing the proper links?

File: boilerplate/app/components/Toggle/Toggle.tsx (hunk 1 above)
Comment (seanpcoyle) [reply boilerplate/app/components/Toggle/Toggle.tsx]: After double-checking, I realized that the `Toggle` component doesn't have an example page, so I removed this line. I did, however, add links to the `Checkbox`, `Radio`, and `Switch` components.

**Review by frankcalise — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**